### PR TITLE
[Notifications] Changed title and moved from MD to HTML table

### PR DESCRIPTION
**Pull request by marciocloudflare**

### CODE REVIEW

File: content/fundamentals/notifications/create-notifications/configure-webhooks.md [hunk 1]
```diff
@@ -17,23 +13,90 @@ This feature is only available if your account has at least one zone on a Profes
...
+   <!-- splunk    -->
+   <tr>
+      <td valign="top"> <a target="_blank" rel="noopener noreferrer" href="https://docs.splunk.com/Documentation/Splunk/latest/Data/UsetheHTTPEventCollector">Splunk</a></td>
+      <td>
+         <p>The secret is required and has to be entered by the user. This is what Splunk refers to as <code>token</code>. Refer to <a target="_blank" rel="noopener noreferrer" href="https://docs.splunk.com/Documentation/Splunk/latest/Data/UsetheHTTPEventCollector#How_the_Splunk_platform_uses_HTTP_Event_Collector_tokens_to_get_data_in">Splunk’s documentation</a> for details.</p>         
+      </td>
+      <td>
+         <p>1. We only support three Splunk endpoints: <code>services/collector</code>, <code>services/collector/raw</code>, and <code>services/collector/event</code>.
+         <p>2. If SSL is enabled on the token, the port must be 443. If SSL is not enabled on the token, the port must be 8088.</p>
+         <p> 3. SSL must be enabled on the server.</p>
```
Comment (pedrosousa) [inline content/fundamentals/notifications/create-notifications/configure-webhooks.md]:
```suggestion
         <p>3. SSL must be enabled on the server.</p>         
```

File: content/fundamentals/notifications/create-notifications/configure-webhooks.md [hunk 2]
```diff
@@ -17,23 +13,90 @@ This feature is only available if your account has at least one zone on a Profes
...
-| [OpsGenie](https://support.atlassian.com/opsgenie/docs/create-a-default-api-integration) | The secret is the `API Key` for OpsGenie's REST API. | `https://api.opsgenie.com/v2/alerts` |
-| [Splunk](https://docs.splunk.com/Documentation/Splunk/latest/Data/UsetheHTTPEventCollector) | The secret is required and has to be entered by the user. This is what Splunk refers to as `token`. Refer to [Splunk’s documentation](https://docs.splunk.com/Documentation/Splunk/latest/Data/UsetheHTTPEventCollector#How_the_Splunk_platform_uses_HTTP_Event_Collector_tokens_to_get_data_in) for details. | 1. We only support three Splunk endpoints: `services/collector`, `services/collector/raw`, and `services/collector/event`. <br/> 2. If SSL is enabled on the token, the port must be 443. If SSL is not enabled on the token, the port must be 8088. <br/> 3. SSL must be enabled on the server. |
-| Generic webhook | User decides. | User decides. |
-
-{{</table-wrap>}}
-
-{{<content-column>}}
+<table>
```
Comment (pedrosousa) [inline content/fundamentals/notifications/create-notifications/configure-webhooks.md:53]: Nit: I usually start how-to pages with the instructions, linking to the references tables included at the end of the page.

File: content/fundamentals/notifications/create-notifications/configure-webhooks.md [hunk 3]
```diff
@@ -17,23 +13,90 @@ This feature is only available if your account has at least one zone on a Profes
...
+    <tr>
+      <th style="width:20%">Service</th>
+      <th style="width:45%">Secret</th>
+      <th style="width:35%">URL</th>
+    </tr>
+  </thead>
+  <tbody>
+   <tr>
+   <!-- google -->
+      <td valign="top"> <a target="_blank" rel="noopener noreferrer" href="https://developers.google.com/chat/how-tos/webhooks">Google Chat</a></td>
```
Comment (pedrosousa) [inline content/fundamentals/notifications/create-notifications/configure-webhooks.md:64]: Nit: I'd rather not have `target` and `rel` attributes hardcoded, but I understand that we don't automatically get these when using HTML.

File: content/fundamentals/notifications/create-notifications/configure-webhooks.md [hunk 4]
```diff
@@ -17,23 +13,90 @@ This feature is only available if your account has at least one zone on a Profes
...
+      <td>
+         <p>The secret is part of the URL. Cloudflare parses this information automatically and there is no input needed from the user.</p>
```
Comment (pedrosousa) [inline content/fundamentals/notifications/create-notifications/configure-webhooks.md]: Nit: I usually leave out paragraphs inside table cells if there's a single paragraph. By including `<p>` tags we're adding a bottom margin to the content, and this makes the table longer.

File: content/fundamentals/notifications/create-notifications/configure-webhooks.md [hunk 5]
```diff
@@ -17,23 +13,90 @@ This feature is only available if your account has at least one zone on a Profes
...
+      </td>
+   </tr>   
+   <!-- splunk    -->
+   <tr>
+      <td valign="top"> <a target="_blank" rel="noopener noreferrer" href="https://docs.splunk.com/Documentation/Splunk/latest/Data/UsetheHTTPEventCollector">Splunk</a></td>
+      <td>
+         <p>The secret is required and has to be entered by the user. This is what Splunk refers to as <code>token</code>. Refer to <a target="_blank" rel="noopener noreferrer" href="https://docs.splunk.com/Documentation/Splunk/latest/Data/UsetheHTTPEventCollector#How_the_Splunk_platform_uses_HTTP_Event_Collector_tokens_to_get_data_in">Splunk’s documentation</a> for details.</p>         
+      </td>
+      <td>
+         <p>1. We only support three Splunk endpoints: <code>services/collector</code>, <code>services/collector/raw</code>, and <code>services/collector/event</code>.
```
Comment (pedrosousa) [inline content/fundamentals/notifications/create-notifications/configure-webhooks.md]:
```suggestion
         <p>1. We only support three Splunk endpoints: <code>services/collector</code>, <code>services/collector/raw</code>, and <code>services/collector/event</code>.</p>
```

File: content/fundamentals/notifications/create-notifications/configure-webhooks.md [hunk 6]
```diff
@@ -17,23 +13,90 @@ This feature is only available if your account has at least one zone on a Profes
...
+      </td>
+      <td>
+         <p>URL varies depending on the Slack channel's address.</p>         
+      </td>
+   </tr>   
+   <!-- datadog    -->
+   <tr>
+      <td valign="top"> <a target="_blank" rel="noopener noreferrer" href="https://docs.datadoghq.com/api/latest/events/#post-an-event">DataDog</a></td>
+      <td>
+         <p>The secret is required and has to be entered by the user. This is what DataDog <a target="_blank" rel="noopener noreferrer" href="https://app.datadoghq.com/account/settings#api">refers to as <code>API Key</code></a>
```
Comment (pedrosousa) [inline content/fundamentals/notifications/create-notifications/configure-webhooks.md]:
```suggestion
         <p>The secret is required and has to be entered by the user. This is what DataDog <a target="_blank" rel="noopener noreferrer" href="https://app.datadoghq.com/account/settings#api">refers to as <code>API Key</code></a>.
```

**Review by pedrosousa — APPROVED** [review]

File: content/fundamentals/notifications/create-notifications/configure-webhooks.md (hunk 3 above)
Comment (marciocloudflare) [reply content/fundamentals/notifications/create-notifications/configure-webhooks.md:64]: Should we not use it then, you think?

File: content/fundamentals/notifications/create-notifications/configure-webhooks.md (hunk 4 above)
Comment (marciocloudflare) [reply content/fundamentals/notifications/create-notifications/configure-webhooks.md]: Done

File: content/fundamentals/notifications/create-notifications/configure-webhooks.md (hunk 2 above)
Comment (marciocloudflare) [reply content/fundamentals/notifications/create-notifications/configure-webhooks.md:53]: Good point. The page evolved and it now makes no sense to have the table here.

**Pull request merged**